.travis.yml

```diff
@@ -12,14 +12,18 @@ before_deploy: |
   if [[ $TRAVIS_HASKELL_VERSION == "8.6.5" ]]; then
     cabal install -f 'static'
     tar -C dist/build/dotenv -cvf dotenv-linux-x86_64-static.tar.gz dotenv
+    curl https://github.com/lindell/github-release-cli/releases/download/1.3.0/github-releaser-travis -L --output github-releaser && chmod +x github-releaser
+    export GITHUB_OAUTH_TOKEN="$GITHUB_TRAVIS_DEPLOY"
+    export RELEASE_NAME=Dotenv-Release-v$(cabal exec dotenv -- --version)
+    export BODY=$(envsubst < ./CHANGELOG.md)
+    export FILES=dotenv-linux-x86_64-static.tar.gz
   fi
 
 deploy:
-  provider: releases
-  api_key: "$GITHUB_TRAVIS_DEPLOY"
-  file: "dotenv-linux-x86_64-static.tar.gz"
+  provider: script
+  script: ./github-releaser -draft -verbose
```

Comment:
@CristhianMotoche taking into account some this script depends on all the operations done at `before_deploy`, maybe we should put everything in one place? That way we also avoid polluting code with conditionals:

```
# file deploy.sh
curl -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/$TRAVIS_REPO_SLUG/releases
```

References: https://docs.travis-ci.com/user/environment-variables/#default-environment-variables